packages/mds-policy/tsconfig.json

```diff
@@ -4,11 +4,26 @@
     "outDir": "dist"
   },
   "references": [
-    { "path": "../../packages/mds-api-server" },
-    { "path": "../../packages/mds-db" },
-    { "path": "../../packages/mds-enums" },
-    { "path": "../../packages/mds-logger" },
-    { "path": "../../packages/mds-providers" },
-    { "path": "../../packages/mds-utils" }
+    {
```

Comment:
Again with the formatting?

Reply:
I figured it out. I'm not sure why yours works still, but for me, disabling `formatOnSave` for json and jsonc files prevents the weird formatting.